docs/en/connector/flink-sql/FlinkEnvConfiguration.md

```diff
@@ -0,0 +1,34 @@
+# Flink Env Configuration
```

Comment:
I don't think this is necessary. Now we can use 'SET parallelism.default = 3' clause to set configuration